gradle.properties

```diff
@@ -0,0 +1,16 @@
+GROUP=com.jakewharton
+VERSION_NAME=7.0.2-SNAPSHOT
```

Comment:
might as well set this to 8.0.0-SNAPSHOT while you're in here! break all the things!